jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js

```diff
@@ -266,29 +266,29 @@ class CreatePaymentUser extends React.Component {
           <div className="create-payment-user__form-content">
             <ul className="create-payment-user__form-type">
               <li className="create-payment-user__form-type-option">
-                <label htmlFor="personal">
-                  <input checked={!this.state.business}
-                    id="personal"
+                <label htmlFor="business">
+                  <input checked={this.state.business}
+                    id="business"
                     name="formType"
-                    onChange={this._setFormType.bind(this, false)}
+                    onChange={this._setFormType.bind(this, true)}
```

Comment:
lets change this so that the _setFormType method either accepts a string, or inspects the input element for the string to apply, 'business' in this case.

Reply:
Is this a show stopper for you?

Reply:
I've tried to do this but it seems like quite a significant refactor?

Reply:
Unless I'm misunderstanding....could this be changed to `onChange={this._setFormType}`

And then the `_setFormType` method would change its contents to something like... 
`this.setState({business: e.target.id === 'business'});`

Reply:
but no it's not a show stopper - just a small improvement